pkg/cluster/member.go

```diff
@@ -139,10 +139,10 @@ func (m *members) store() {
 func (m *members) self() *member {
 	m.RLock()
 	defer m.RUnlock()
-	return m._self()
+	return m.unsafeSelf()
```

Comment:
Actually, the member feature was deprecated and should be deleted.

Reply:
Yes, but it requires some efforts. And the code changes in this PR are not as much as it looks, most code changes are a result of moving a function from one file to another.